index.html

```diff
@@ -0,0 +1,201 @@
+                    </defs>
+                </svg>
+                <div class="logo-img">
+                    <img src="/images/logo.png">
+                </div>
+            </div>
+            <div class="total-desktop-heading">
+                <div class="header">
+                    <ul>
+                        <li><img src="/images/Rectangle.svg"></li>
+                        <li><img src="/images/twitter.svg"></li>
+                        <li><a>English</a></li>
+                        <li><a>My page</a></li>
+                        <li><a>logout</a></li>
+                    </ul>
+                </div>
+                <div class="heading-desktop">
+                    <div class="logo-img">
+                        <img src="/images/logo.png">
+                    </div>
+                    <ul class="desktop-lists">
+                        <li class="about">
+                            <a href="/html/about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="#program">Program</a>
+                        </li>
+                        <li>
+                            <a href="#join">Join</a>
+                        </li>
+                        <li>
+                            <a href="#sponsor">sponsor</a>
+                        </li>
+                        <li>
+                            <a href="#news">news</a>
+                        </li>
+                        <li class="campaign">
+                            <a href="#cc campaign">campaign</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <section class="hero">
+        <div class="hero-text main-hero-txt">
+            <div class="hero-headings ">
+               <div class="first-part first-part-hero">
+                <p>"Hello! Sharing  world"</p>
+                <h1>CREATIVE COMMONS GLOBAL SUBMIT 2015</h1>
+               </div>
+                <p class="second-part second-part-hero ">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+            <div class="date">
+                <h3 class="date-head">2015.10.15(THU) ~ 16(FRI)</h3>
+                <p class="date-second">@ National Museum of Karea, Art Center Nabi and more</p>
+            </div>
+        </div>
+    </section>
```

Comment:
- Please be sure to stick to the original design, which includes:
   - [ ] Increasing the font size to match the given template.
   - [ ] Reducing the space between the elements to match the given design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/olanikegloria/capstone-project/assets/62290545/3fad6d8b-5ad6-465a-8ec4-6ca72e16e84c) | ![image](https://github.com/olanikegloria/capstone-project/assets/62290545/3b565766-c2de-4ab0-9109-59fdbd5cdb9a) |